File: 8KYU/UEFA EURO 2016/solution.cs
public class Kata
{
  public static string UefaEuro2016(string[] teams, int[] scores)
  {
    if(scores[0] == scores[1])
      return $"At match {teams[0]} - {teams[1]}, teams played draw.";
    else if(scores[0] > scores[1])
      return $"At match {teams[0]} - {teams[1]}, {teams[0]} won!";
    else
      return $"At match {teams[0]} - {teams[1]}, {teams[1]} won!";
  }
}

File: 8KYU/UEFA EURO 2016/solution.js
function uefaEuro2016(teams, scores){
    if(scores[1] > scores[0])
      return `At match ${teams[0]} - ${teams[1]}, ${teams[1]} won!`;
    else if(scores[0] > scores[1])
      return `At match ${teams[0]} - ${teams[1]}, ${teams[0]} won!`;
    else
      return `At match ${teams[0]} - ${teams[1]}, teams played draw.`;
  }
  
  // "At match Germany - Ukraine, Germany won!"
  // "At match Portugal - Iceland, teams played draw."